src/sentry/static/sentry/app/icons/icon-warning-sm.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Icon from 'react-icon-base';
+
+function IconWarningSm(props) {
```

Comment:
we should consider some abstraction across these components so they inherit from the same thing maybe, to make future modification easier. probably not necessary yet